app/Models/Playlist.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+use Spatie\MediaLibrary\HasMedia;
+use Spatie\MediaLibrary\InteractsWithMedia;
+
+class Playlist extends Model implements HasMedia
+{
+    use HasFactory;
+    use InteractsWithMedia;
+
+    public function getImageAttribute()
+    {
+        return $this->getFirstMediaUrl();
+    }
+
+    public $with = [
```

Comment:
Maybe put this above the attribute getter

Reply:
Good catch!